sphinx-doc/package-hoomd.rst

```diff
@@ -31,13 +31,11 @@ hoomd
 .. toctree::
    :maxdepth: 3
 
-   module-hoomd-analyze
    module-hoomd-array
    module-hoomd-comm
    module-hoomd-custom
    module-hoomd-data
```

Comment:
With type parameters and friends moved here, we should document them in module-hoomd-data.rst instead of where they are now (I think it is in misc).